vuepress/docs/v7.0/docs/reference/databases.md

```diff
@@ -84,82 +74,89 @@ The EntandoDatabaseService custom resource looks like this:
       tablespace: (Oracle only)  the tablespace to use to create the required schemas in
       jdbcParameters: a map containing name-value pairs for any additional parameters required for the JDBC driver to connect to the database.
 
-The Secret that will provide the admin credentials, identified by the
-above `secretName` should look like this:
+An example of the Secret that provides the admin credentials, identified as
+`secretName` above:
 
     Secret:
       name: string, any K8S compliant name
-      namespace: string, the namespace this will be created in
+      namespace: string, the namespace the DB is created in
       stringData:
         username: string, name of an admin user that can create schemas and other users
         password: string, password of the above user
 
-### How it works
-
-In order for the EntandoApp and EntandoPlugin deployer to pick up the
-correct database service, the EntandoDatabaseService needs to be created
-BEFORE the EntandoApps and EntandoPlugins are created. There can be
-multiple EntandoDatabaseServices in the namespace, but they need to
-point to database services of different vendors, i.e. PostgreSQL, Oracle
-and MySQL. Entando currently cannot enforce any validation but if there
-are two EntandoDatabaseServices that have the same DBMS vendor, it will
+### How It Works
+#### Database Custom Resource
+In order for the EntandoApp and Plugin deployer to choose the
```

Comment:
is it a single deployer? why is deployer not capitalized? i feel like this should read "EntandoApp and plugin deployer"

Reply:
There's a s single deployer job per plugin which does its work and then shuts down.